packages/x-data-grid-premium/src/hooks/features/aggregation/gridAggregationInterfaces.ts

```diff
@@ -74,6 +81,36 @@ export interface GridAggregationFunction<V = any, AV = V> {
   getCellValue?: (params: GridAggregationGetCellValueParams) => V;
 }
 
+/**
+ * Grid aggregation function data sourcedefinition interface.
+ * @demos
+ *   - [Server-side aggregation](/x/react-data-grid/server-side-data/aggregation/)
+ */
+export interface GridAggregationFunctionDataSource {
```

Comment:
you can make `GridAggregationFunction` extend this interface to keep the shared props aligned

Reply:
Nice idea, however I'd prefer the other way around, i.e.`Omit<GridAggregationFunction, 'apply' | 'getCellValue'>` to not alter the existing interface.